src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((arr) => arr.split(':'))
+    .filter((item) => item.length > 1)
+    .reduce((i, item) => {
```

Comment:
[CODE KNOWLEDGE] - Don't use iteration methods to modify an already existing array/object. These methods return a new array, so you should use it. Here, `reduce` is being used correctly to build a new object, but the parameter `i` is not descriptive. Consider renaming it to something like `styleObject` to improve readability.